chrome/content/zotero/xpcom/data/item.js

```diff
@@ -1274,7 +1274,7 @@ Zotero.Item.prototype._saveData = Zotero.Promise.coroutine(function* (env) {
 		yield Zotero.DB.queryAsync(sql, env.sqlValues);
 		
 		if (!env.options.skipNotifier) {
-			Zotero.Notifier.queue('modify', 'item', itemID, env.notifierData);
+			Zotero.Notifier.queue('modify', this._objectType, itemID, env.notifierData);
```

Comment:
Hmm, is this right? I would expect various things (e.g., in itemTreeView) not to work properly if a 'feedItem' event is passed instead of 'item'. But maybe it also avoids things we don't want?